<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BendDialog</class>
 <widget class="QDialog" name="BendDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bend</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="bendTypeLabel">
       <property name="text">
        <string>Bend Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="bendTypeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="bentPitchLabel">
       <property name="text">
        <string>Bent Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="bentPitchComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="releasePitchLabel">
       <property name="text">
        <string>Release Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="releasePitchComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="bendDurationLabel">
       <property name="text">
        <string>Bend Duration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="defaultDurationButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QRadioButton" name="customDurationButton">
           <property name="text">
            <string>Current note duration + next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="bendDurationSpinBox"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>notes</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="vertStartingPointLabel">
       <property name="text">
        <string>Vertical Starting Point:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="vertStartingPointComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vertEndingPointLabel">
       <property name="text">
        <string>Vertical Ending Point:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="vertEndingPointComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
